src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -147,6 +151,84 @@ const char* GetCalendarName(CalendarId calendarId)
 
 /*
 Function:
+GetCalendarName
+
+Gets the associated CalendarId for the ICU calendar name.
+*/
+CalendarId GetCalendarId(const char* calendarName)
+{
+	if (strcmp(calendarName, "gregorian") == 0)
```

Comment:
The strings that get passed into here come from CLDR.  So it is possible that someone put in a different casing - and looking at the ICU code, they are using an insensitive comparison when creating calendars.  Will update to case insensitive.